modules/services/window-managers/herbstluftwm.nix

```
@@ -119,6 +119,22 @@ in
         {file}`$XDG_CONFIG_HOME/herbstluftwm/autostart`.
       '';
     };
+
+    herbstclientAlias = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
```

```suggestion
    enableAlias = lib.mkOption {
      type = lib.types.bool;
      default = true;
```

Already in the module so it feels redundant and it follows convention used elsewhere.